spec/features/node_pages/notes_table_toolbar_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+describe 'node pages' do
+  describe '#show notes table toolbar', js: true do
+    subject { page }
+
+    before do
+      login_to_project_as_user
+
+      @node  = create(:node)
+      @notes = []
+      (Note::MAX_DELETED_INLINE + 1).times do |i|
+        @notes << create(:note, node: @node, text: "#[Title]#\r\ntest#{i}\r\n\r\n#[Description]#\r\nnote#{i}\r\n")
+      end
+
+      visit node_path(@node, tab: 'notes-tab')
+    end
+
+    context 'when Select All is clicked' do
+      before do
+        find('.js-index-table-select-all').click
+      end
+
+      it 'selects all notes' do
+        all('input[type=checkbox].js-multicheck').each do |el|
+          expect(el['checked']).to be true
+        end
+      end
+
+      it 'shows the note actions bar' do
+        expect(find('.js-index-table-actions')).to be_visible
+      end
+    end
+
+    describe 'when clicking notes' do
+      it 'displays action buttons (delete button) if 1 note is clicked' do
+        expect(find('.js-index-table-actions', visible: :all)).to_not be_visible
+        check "checkbox_note_#{@notes[0].id}"
+        expect(find('.js-index-table-actions')).to be_visible
+      end
+
+      # context 'deleting with background job' do
```

Comment:
what is this commented code for?

Reply:
I have a problem with sqlite and the background process here, have to take a deeper look into it